localstack/services/stepfunctions/asl/eval/environment.py

```diff
@@ -26,8 +26,14 @@
 
 
 class Environment:
-    def __init__(self, context_object_init: ContextObjectInitData):
+    def __init__(
+        self, account_id: str, region_name: str, context_object_init: ContextObjectInitData
+    ):
         super(Environment, self).__init__()
+
```

Comment:
I think we should decorate these in an object where the environment can expose runtime values like these 2.